manager/controllers/app/modules/moduleinterface.go

```diff
@@ -110,12 +110,19 @@ func (m *Selector) SupportsInterface(module *app.FybrikModule, requestedCapabili
 			}
 		} else if requestedCapability == app.Copy {
 			for _, inter := range capability.SupportedInterfaces {
+				if inter.Source == nil {
+					break
```

Comment:
It would be better to continue to check another interface, s.t. we won't miss a valid entry.

Reply:
Resolved.